app/src/main/java/com/rafsan/newsapp/ui/feed/FeedFragment.kt

```diff
@@ -107,7 +106,7 @@ class FeedFragment : BaseFragment<FragmentFeedBinding>() {
     }
 
     private fun setupObservers() {
-        lifecycleScope.launchWhenStarted {
+        viewLifecycleOwner.lifecycleScope.launchWhenStarted {
```

Comment:
```viewLifecycleOwner.lifecycleScope``` this causes the result to be not collected when clear out of search.

For example: 
Search for a news topic->Click item to view detail->Go back to home page->Close search-> Initial news items are not showing (Search result is still showing).

This is due to ```viewLifecycleOwner.lifecycleScope``` coroutine gets suspended.